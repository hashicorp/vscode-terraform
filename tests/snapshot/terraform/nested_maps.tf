ipam_configuration = {
  us-east-1 = {
    cidr   = ["10.0.0.0/16"]
    locale = "us-east-1"

    sub_pools = {
      dev = {
        cidr                              = ["10.0.0.0/20"]
        allocation_default_netmask_length = 28

        sub_pools = {
          app_team_b = {
            cidr        = ["10.0.0.0/24"]
            description = "app team b's space in us-east-1"
          }
          app_team_a = {
            cidr = ["10.0.1.0/24"]
          }
        }
      }

      stage = {
        cidr = ["10.0.16.0/20"]
      }
    }
  }
}
