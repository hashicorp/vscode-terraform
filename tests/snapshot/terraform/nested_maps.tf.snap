>ipam_configuration = {
#^^^^^^^^^^^^^^^^^^ source.terraform variable.declaration.terraform variable.other.readwrite.terraform
#                  ^ source.terraform variable.declaration.terraform
#                   ^ source.terraform variable.declaration.terraform keyword.operator.assignment.terraform
#                    ^ source.terraform variable.declaration.terraform
#                     ^ source.terraform meta.braces.terraform punctuation.section.braces.begin.terraform
>  us-east-1 = {
#^^ source.terraform meta.braces.terraform
#  ^^^^^^^^^ source.terraform meta.braces.terraform meta.mapping.key.terraform variable.other.readwrite.terraform
#           ^ source.terraform meta.braces.terraform
#            ^ source.terraform meta.braces.terraform keyword.operator.assignment.terraform
#             ^ source.terraform meta.braces.terraform
#              ^ source.terraform meta.braces.terraform meta.braces.terraform punctuation.section.braces.begin.terraform
>    cidr   = ["10.0.0.0/16"]
#^^^^ source.terraform meta.braces.terraform meta.braces.terraform
#    ^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.mapping.key.terraform variable.other.readwrite.terraform
#        ^^^ source.terraform meta.braces.terraform meta.braces.terraform
#           ^ source.terraform meta.braces.terraform meta.braces.terraform keyword.operator.assignment.terraform
#            ^ source.terraform meta.braces.terraform meta.braces.terraform
#             ^ source.terraform meta.braces.terraform meta.braces.terraform
#              ^ source.terraform meta.braces.terraform meta.braces.terraform string.quoted.double.terraform punctuation.definition.string.begin.terraform
#               ^^^^^^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform string.quoted.double.terraform
#                          ^ source.terraform meta.braces.terraform meta.braces.terraform string.quoted.double.terraform punctuation.definition.string.end.terraform
#                           ^^ source.terraform meta.braces.terraform meta.braces.terraform
>    locale = "us-east-1"
#^^^^ source.terraform meta.braces.terraform meta.braces.terraform
#    ^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.mapping.key.terraform variable.other.readwrite.terraform
#          ^ source.terraform meta.braces.terraform meta.braces.terraform
#           ^ source.terraform meta.braces.terraform meta.braces.terraform keyword.operator.assignment.terraform
#            ^ source.terraform meta.braces.terraform meta.braces.terraform
#             ^ source.terraform meta.braces.terraform meta.braces.terraform string.quoted.double.terraform punctuation.definition.string.begin.terraform
#              ^^^^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform string.quoted.double.terraform
#                       ^ source.terraform meta.braces.terraform meta.braces.terraform string.quoted.double.terraform punctuation.definition.string.end.terraform
>
>    sub_pools = {
#^^^^ source.terraform meta.braces.terraform meta.braces.terraform
#    ^^^^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.mapping.key.terraform variable.other.readwrite.terraform
#             ^ source.terraform meta.braces.terraform meta.braces.terraform
#              ^ source.terraform meta.braces.terraform meta.braces.terraform keyword.operator.assignment.terraform
#               ^ source.terraform meta.braces.terraform meta.braces.terraform
#                ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform punctuation.section.braces.begin.terraform
>      dev = {
#^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#      ^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.mapping.key.terraform variable.other.readwrite.terraform
#         ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#          ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform keyword.operator.assignment.terraform
#           ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#            ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform punctuation.section.braces.begin.terraform
>        cidr                              = ["10.0.0.0/20"]
#^^^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#        ^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.mapping.key.terraform variable.other.readwrite.terraform
#            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#                                          ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform keyword.operator.assignment.terraform
#                                           ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#                                            ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#                                             ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform string.quoted.double.terraform punctuation.definition.string.begin.terraform
#                                              ^^^^^^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform string.quoted.double.terraform
#                                                         ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform string.quoted.double.terraform punctuation.definition.string.end.terraform
#                                                          ^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
>        allocation_default_netmask_length = 28
#^^^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.mapping.key.terraform variable.other.readwrite.terraform
#                                         ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#                                          ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform keyword.operator.assignment.terraform
#                                           ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#                                            ^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform constant.numeric.integer.terraform
>
>        sub_pools = {
#^^^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#        ^^^^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.mapping.key.terraform variable.other.readwrite.terraform
#                 ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#                  ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform keyword.operator.assignment.terraform
#                   ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#                    ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform punctuation.section.braces.begin.terraform
>          app_team_b = {
#^^^^^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#          ^^^^^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.mapping.key.terraform variable.other.readwrite.terraform
#                    ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#                     ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform keyword.operator.assignment.terraform
#                      ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#                       ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform punctuation.section.braces.begin.terraform
>            cidr        = ["10.0.0.0/24"]
#^^^^^^^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#            ^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.mapping.key.terraform variable.other.readwrite.terraform
#                ^^^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#                        ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform keyword.operator.assignment.terraform
#                         ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#                          ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#                           ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform string.quoted.double.terraform punctuation.definition.string.begin.terraform
#                            ^^^^^^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform string.quoted.double.terraform
#                                       ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform string.quoted.double.terraform punctuation.definition.string.end.terraform
#                                        ^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
>            description = "app team b's space in us-east-1"
#^^^^^^^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#            ^^^^^^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.mapping.key.terraform variable.other.readwrite.terraform
#                       ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#                        ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform keyword.operator.assignment.terraform
#                         ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#                          ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform string.quoted.double.terraform punctuation.definition.string.begin.terraform
#                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform string.quoted.double.terraform
#                                                          ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform string.quoted.double.terraform punctuation.definition.string.end.terraform
>          }
#^^^^^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#          ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform punctuation.section.braces.end.terraform
>          app_team_a = {
#^^^^^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#          ^^^^^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.mapping.key.terraform variable.other.readwrite.terraform
#                    ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#                     ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform keyword.operator.assignment.terraform
#                      ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#                       ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform punctuation.section.braces.begin.terraform
>            cidr = ["10.0.1.0/24"]
#^^^^^^^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#            ^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.mapping.key.terraform variable.other.readwrite.terraform
#                ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#                 ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform keyword.operator.assignment.terraform
#                  ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#                   ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#                    ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform string.quoted.double.terraform punctuation.definition.string.begin.terraform
#                     ^^^^^^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform string.quoted.double.terraform
#                                ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform string.quoted.double.terraform punctuation.definition.string.end.terraform
#                                 ^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
>          }
#^^^^^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#          ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform punctuation.section.braces.end.terraform
>        }
#^^^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#        ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform punctuation.section.braces.end.terraform
>      }
#^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#      ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform punctuation.section.braces.end.terraform
>
>      stage = {
#^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#      ^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.mapping.key.terraform variable.other.readwrite.terraform
#           ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#            ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform keyword.operator.assignment.terraform
#             ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#              ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform punctuation.section.braces.begin.terraform
>        cidr = ["10.0.16.0/20"]
#^^^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#        ^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.mapping.key.terraform variable.other.readwrite.terraform
#            ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#             ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform keyword.operator.assignment.terraform
#              ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#               ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#                ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform string.quoted.double.terraform punctuation.definition.string.begin.terraform
#                 ^^^^^^^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform string.quoted.double.terraform
#                             ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform string.quoted.double.terraform punctuation.definition.string.end.terraform
#                              ^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
>      }
#^^^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#      ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform punctuation.section.braces.end.terraform
>    }
#^^^^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform
#    ^ source.terraform meta.braces.terraform meta.braces.terraform meta.braces.terraform punctuation.section.braces.end.terraform
>  }
#^^ source.terraform meta.braces.terraform meta.braces.terraform
#  ^ source.terraform meta.braces.terraform meta.braces.terraform punctuation.section.braces.end.terraform
>}
#^ source.terraform meta.braces.terraform punctuation.section.braces.end.terraform
>